### ISSUE TITLE
NTR: 21 deoxycortisol high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8367]
New term request 21 deoxycortisol high in blood (elthat_131007043844):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000516
- label: Elevated circulating 21-deoxycortisol concentration
- parents: [Increased circulating cortisol level (HP:0003118)](https://hpo.jax.org/app/browse/term/HP:0003118)
- pmid: PMID:34780778
- def: The concentration of 21-deoxycortisol in the blood circulation is above the upper limit of normal.
- comment: 21-Deoxycortisol is a marker for 21-hydroxylase deficiency.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8367
- synonyms:
